app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Can we move to `es6` for stronger typing?

Reply:
yes! I wonder why it nextJS chose es5 by default 🤔